### DISCUSSION TITLE
Git Required when signing/deploying artifacts?

### DISCUSSION THREAD

**Discussion by gastaldi** [jreleaser/jreleaser#1710] [Q&A] [answered]
While running JReleaser, I noticed that it takes some info from Git, for example, if I don't have a remote it fails with: 

```bash
$ jreleaser sign                                                                                                           
[INFO]  JReleaser 1.13.1
[INFO]  Configuring with jreleaser.yml
[INFO]    - basedir set to /private/tmp/jreleaser
[INFO]    - outputdir set to /private/tmp/jreleaser/out/jreleaser
[INFO]  Reading configuration
[INFO]  git-root-search set to false
Could not determine remote
repository doesn't have a remote named 'origin'
```

In my scenario, the Git repository I'm working on is not related to the project being deployed, is there a way to turn any check to Git off?

**Comment by aalmiray** [comment, marked as answer]
Git checks are disabled for some commands such as `assemble` and `deploy`. Perhaps this check should be skipped for `sign` and maybe a few other commands such as `catalog`, `checksum`, and `upload`.

**Reply by aalmiray** [reply]
Filed https://github.com/jreleaser/jreleaser/issues/1713
